modules/memberships/class-jetpack-memberships.php

```diff
@@ -0,0 +1,291 @@
+<?php
+/**
```

Comment:
I think you can remove that comment altogtether. Once it's in Fusion there will be a pre-commit warning when you make changes to the file on WordPress.com anyway.

Reply:
Done!